src/Handler/IteratorHandler.php

```diff
@@ -25,22 +25,6 @@ public static function getSubscribingMethods()
     {
         $methods = [];
 
-        foreach (self::SUPPORTED_FORMATS as $format) {
-            $methods[] = [
-                'direction' => GraphNavigatorInterface::DIRECTION_SERIALIZATION,
-                'type' => iterable::class,
```

Comment:
iterable is a psydo class and does not really exist. this is resolved as a class in the current namespace, i.e. `JMS\Serializer\Handler\iterable` which does not exist. this is why phpstan complained rightfully.
also the whole registration of `iterable` is pointless because iterables are already handled directly in the graph visitor (like for array, int etc)

Reply:
why not put just `'iterable'` as string?

Reply:
Because it wouldn't do anything? iterables are not handled via registration.